js/ui/apple2.ts

```diff
@@ -756,12 +759,12 @@ export function updateUI() {
     }
 }
 
-export function pauseRun() {
+export async function pauseRun() {
     const label = document.querySelector<HTMLElement>('#pause-run i')!;
     if (paused) {
         _apple2.ready.then(() => {
             _apple2.run();
-        });
+        }).catch(console.error);
```

Comment:
Since the function is now `async`, consider:
```typescript
if (paused) {
  try {
    await _apple2.ready();
    _apple2.run();
  } catch (e) {
    console.error(e);
  }
  ...
```
otherwise the labels will be changed before `ready` actually completes.

Reply:
Oops, I didn't mean to make this async. It's not clear that async and event handlers go well together just yet.